spec/features/card_check_translation_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+describe 'Check user translation' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.